src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/POSTHyperlinkNote/script.js

```diff
@@ -0,0 +1,8 @@
+Behaviour.specify("A.post-hyperlink-note-button", "POSTHyperLinkNote", 0, function (element) {
```

Comment:
The magic behind this file is https://github.com/jenkinsci/jenkins/blob/dc4ad7e1a872e6dd9504f6c1ff1f95d84ca6046b/core/src/main/java/hudson/console/ConsoleAnnotationDescriptor.java#L84-L86.